assets/src/components/editing/editor/normalizers/normalizer.ts

```diff
@@ -14,15 +16,62 @@ export interface NormalizerContext {
 
 const restrictedElements = new Set(['input_ref']);
 
-export function installNormalizer(editor: Editor, context: NormalizerContext = {}) {
```

Comment:
Added the ability to configure what normalizers were configured, only needed 2 of these options for this, but made them all to be complete.